### PR TITLE
repoHandle plugins

### PR DESCRIPTION
**Pull request by xwz123**
Complete gitee warehouse creation plug-in migration

### CODE REVIEW

File: prow/cmd/gitee-hook/BUILD.bazel [hunk 1]
```diff
@@ -68,6 +68,7 @@ go_library(
         "//prow/gitee-plugins/lgtm:go_default_library",
         "//prow/gitee-plugins/slack:go_default_library",
         "//prow/gitee-plugins/trigger:go_default_library",
+	"//prow/gitee-plugins/repohandle:go_default_library",
```
Comment (zengchen1024) [inline prow/cmd/gitee-hook/BUILD.bazel:71]: fix the indent

File: prow/cmd/gitee-hook/main.go [hunk 2]
```diff
@@ -223,7 +229,7 @@ func buildClients(o *options, secretAgent *secret.Agent, pluginAgent *plugins.Co
 		giteeClient:    giteeClient,
 		giteeGitClient: giteeGitClient,
 		ownersClient:   ownersClient,
-		prowJobClient:  prowJobClient,
+		//prowJobClient:  prowJobClient,
```
Comment (zengchen1024) [inline prow/cmd/gitee-hook/main.go:232]: why delete it

File: prow/cmd/gitee-hook/plugins.go [hunk 3]
```diff
@@ -8,21 +8,22 @@ import (
...
-func initPlugins(cfg prowConfig.Getter, agent *plugins.ConfigAgent, pm plugins.Plugins, cs *clients) error {
+func initPlugins(cfg prowConfig.Getter, agent *plugins.ConfigAgent, pm plugins.Plugins, cs *clients) ([]plugins.Plugin,error)  {
 	gpc := func(name string) plugins.PluginConfig {
 		return agent.Config().GetPluginConfig(name)
 	}
 
 	botname, err := cs.giteeClient.BotName()
 	if err != nil {
-		return err
+		return nil,err
```
Comment (zengchen1024) [inline prow/cmd/gitee-hook/plugins.go:26]: fix the format. there is a whitespace between nil and err

Comment (xwz123) [reply prow/cmd/gitee-hook/plugins.go]: already fixed

File: prow/cmd/gitee-hook/BUILD.bazel (hunk 1 above)
Comment (xwz123) [reply prow/cmd/gitee-hook/BUILD.bazel]: already fixed

**Pull request closed without merging**